lib/core/utils/is-hidden.js

```diff
@@ -15,6 +15,14 @@ axe.utils.isHidden = function isHidden(el, recursed) {
 		return false;
 	}
 
+	/**
+	 * Handle media elements that can be hidden by the absence of an attribute
+	 * Eg: `audio` without `controls` attribute is hidden by default,
+	 */
+	if (el.nodeName.toUpperCase() === `AUDIO` && !el.hasAttribute('controls')) {
+		return false;
```

Comment:
This should return true as the element is hidden

Reply:
We are trying to negate what is by default hidden.

`excludeHidden` is a flag we have on rules to aggregate the nodes necessary for each rule to run. By default excludeHidden is true. When excludeHidden is true, the isHidden fn is called to filter the nodes.

We only have the option to say `excludeHidden: false` in a rule's config, which basically means the nodes are not filtered via the isHidden fn.

Here we are manipulating HTMLMediaElement `audio` which by default is hidden when there is no controls attribute in browsers, we want that node to run through our check, so we want this to return false, so it gets added as a candidate for the check.